server/routerlicious/packages/routerlicious/src/deli/index.ts

```diff
@@ -40,6 +40,12 @@ export async function deliCreate(config: Provider): Promise<core.IPartitionLambd
     // Database connection for global db if enabled
     const factory = await services.getDbFactory(config);
 
+    const checkpointHeuristics = config.get("deli:checkpointHeuristics") as
+                            IDeliCheckpointHeuristicsServerConfiguration;
+    if (checkpointHeuristics && checkpointHeuristics.enable) {
+        core.DefaultServiceConfiguration.deli.checkpointHeuristics = config.get("deli:checkpointHeuristics");
```

Comment:
looks like in line 43 the result of `config.get("deli:checkpointHeuristics")` is assigned to checkpointHeuristics. Could we reuse checkpointHeuristics here (when setting `core.DefaultServiceConfiguration.deli.checkpointHeuristics`) instead of calling `config.get()` again?

Reply:
yeah, looks like that change wasn't staged. Added it now.